tp2/Pacmania/Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class GameManager : MonoBehaviour {
+
+	public static GameManager instance = null;				// Static instance of GameManager which allows it to be accessed by any other script.
+	[HideInInspector] public int score;
+	[HideInInspector] public int[,] board;
+	private int lives;
+
+
+	void Awake() {
+		//Check if instance already exists
+		if (instance == null)
+		{
+			//if not, set instance to this
+			instance = this;
+		}
+		//If instance already exists and it's not this:
+		else if (instance != this)
+		{
+			//Then destroy this. This enforces our singleton pattern, meaning there can only ever be one instance of a GameManager.
+			Destroy (gameObject);	
+		}
+		// We need the scores at the next scene, so we store them here.
+		DontDestroyOnLoad(gameObject);
+
+		//Call the InitGame function to initialize the first level 
+		InitGame();
+	}
+
+	// Use this for initialization
+	void Start () {
+		score = 0;
+
+	}
+	
+	// Update is called once per frame
+	void Update () {
+	
+	}
+
+	void InitGame() {
+		score = 0;
```

Comment:
Done! I let the one inside InitGeme, and remove the definition inside the Start function.

We have to discuss this later, if we implements levels maybe this wouldnt be the place to define the score and the lives.